### DISCUSSION TITLE
Logo proposal

### DISCUSSION THREAD

**Discussion by simonsan** [rustic-rs/rustic#618] [Ideas]
Generated some by midjourney:

![simonsan_rust_crab_ferris_umbrella_huge_claws_rain_background_t_022745ef-68cd-4c8a-9cde-4cbddf9290d0](https://user-images.githubusercontent.com/14062932/235012315-8334fa5e-2eeb-43ea-8ddb-8edc88abfc50.png)
![simonsan_rust_crab_ferris_umbrella_huge_claws_rain_simple_vecto_27e25ae4-696a-4bde-81e1-dd6894f2af1e](https://user-images.githubusercontent.com/14062932/235012318-af85157a-b50b-40a8-a8a5-cbe2bb13fb4a.png)
![simonsan_rust_crab_ferris_umbrella_huge_claws_rain_simple_vecto_b343706a-9f5e-45dd-a687-7716aa1cc0e4](https://user-images.githubusercontent.com/14062932/235012320-37bf2bd9-14f0-42b4-90e8-68f7054f727c.png)
![simonsan_rust_crab_ferris_umbrella_huge_claws_rain_simple_vecto_239fb8b3-fe0c-4a9e-ad58-98709cc8c0ee](https://user-images.githubusercontent.com/14062932/235012321-1dd38a08-84e9-4792-9ee7-96eb05676753.png)
![simonsan_rust_crab_ferris_umbrella_huge_claws_rain_simple_vecto_9f500997-5e86-4d11-971c-01a79af3681c](https://user-images.githubusercontent.com/14062932/235012322-42022f1e-3dbf-4298-8d9d-4c77c36a9daf.png)
![simonsan_rust_crab_ferris_umbrella_huge_claws_rain_simple_vecto_4f168ee0-4ff6-42fb-b474-1d3ab197d01c](https://user-images.githubusercontent.com/14062932/235012325-c6bf7ca7-62f1-4e29-9d57-5bc522e95532.png)
![simonsan_rust_crab_ferris_friendly_umbrella_huge_claws_rain_in__2c7fe5fb-44ea-41c6-bbb2-9e02fa3038ff](https://user-images.githubusercontent.com/14062932/235012615-d0fbb69f-6d3a-42d1-91ba-0f90bf79929e.png)

**Comment by simonsan** [comment]
![simonsan_realistic_rust_mascot_crab_named_ferris_holding_umbrel_f71112c2-a4cc-44d2-aba8-7fa3330b5afc](https://user-images.githubusercontent.com/14062932/235013236-a2805084-d7d6-4f4b-8845-c0a064c9757a.png)

**Comment by aawsome** [comment]
Sorry for coming up late to this topic. I very much like the idea and your proposals.
What about turning this into a poll and define a timeline for users to vote for one of proposals?

**Reply by simonsan** [reply]
I'm thinking, maybe we should make a call for participation on that one, so others are also proposing their ideas and know about it? I'm sure that there need to be better ones that these generated ones.

**Comment by LeoniePhiline** [comment]
The first one's style rocks! What's your prompt?

Opinion:

I find the crustacean a little evil-looking in many of the proposals.

Ferris is not part of the Rust trademark and can be used. How about using ferris rather than an anonymous or even evil looking crab?

**Reply by simonsan** [reply]
Prompt

```text
rust crab ferris, umbrella, huge claws, rain, background trees, cozy, simple, vector --no shading detail --v 5
```

"Evil looking":
```text
rust crab ferris, umbrella, huge claws, rain, simple, vector --no shading detail --v 5
```

```text
realistic rust mascot crab named ferris, holding umbrella, friendly look, huge claws, rain, in the woods, protective, simple, vector --no shading detail --v 5
```

```text
rust mascot crab ferris, holding umbrella, friendly look, huge claws, rain, in the woods, protective, simple, vector --no shading detail --v 5
```

**Reply by simonsan** [reply]
And I agree, they are all not totally what I wanted. Just wanted to give some material to get going. ;)

P.S.: Hello, fellow Berliner :)

**Reply by simonsan** [reply]
I wanted to imitate the `restic` logo with a Rust one, if that was unclear:

![logo 1](https://github.com/rustic-rs/rustic/assets/14062932/80c3cb47-a2d4-4e0e-b3d6-c6365688811e)

**Reply by simonsan** [reply]
There might be better results possible by using ChatGPT to let it generate a prompt for Midjourney (as absurd as it sounds 😅 ):
https://github.com/chatgpt-prompts/ChatGPT-Midjourney-Prompt-Generator

**Comment by simonsan** [comment]
@LeoniePhiline @aawsome

Prompt: `In a heartwarming vector illustration, Ferris, Rust's cherished mascot, finds solace under an umbrella. Rain envelops the scene, but Ferris remains untouched, its massive claws gripping the umbrella's handle. The crab's friendly eyes and cheerful demeanor radiate warmth, making it the focal point amidst the rain. Behind Ferris, the forest stands majestic, its dense canopy adding depth and texture to the scene. The droplets on the trees and the wet ground only accentuate the feeling of a rainy day. Yet, amidst this, the composition offers a profound sense of security, making one feel sheltered just like Ferris. --v 5.2` (generated by ChatGPT-Midjourney-Prompt-Generator)

![0_1](https://github.com/rustic-rs/rustic/assets/14062932/3f5cef95-2782-4af5-ba2e-4a2922229ef3)
![0_2](https://github.com/rustic-rs/rustic/assets/14062932/bcb65ea7-24f8-4b82-b2bc-70ff9b784e9f)

**Reply by LeoniePhiline** [reply]
The prompt alone is such poetry. 😺

**Reply by simonsan** [reply]
Which one of these do you prefer?

**Reply by aawsome** [reply]
I like those friendly-looking crabs much more! The first one is my personal favorite.

**Comment by simonsan** [comment]
![rustic_time_capsule](https://github.com/rustic-rs/rustic/assets/14062932/f049d9ae-2d1d-4da8-9848-a7db4c100abc)
![rustic_time_capsule_edge](https://github.com/rustic-rs/rustic/assets/14062932/e0f4cd33-d2fb-4b25-8631-a9ffec1500ad)

**Comment by simonsan** [comment]
@LeoniePhiline @aawsome Opinions?

![Logo_new](https://github.com/rustic-rs/rustic/assets/14062932/ed9996e3-a505-4470-ad4b-fdeb42fee57d)
![Logo_new_white](https://github.com/rustic-rs/rustic/assets/14062932/09f58a7d-9157-46d6-8f26-5f2f3a7bf9c0)

**Reply by LeoniePhiline** [reply]
I love it

**Comment by simonsan** [comment]
For Discord:
![Logo_new](https://github.com/rustic-rs/rustic/assets/14062932/62bfabb2-2904-478c-b341-c2afcc6368f9)

General, e.g. readme?
![Logo_new1](https://github.com/rustic-rs/rustic/assets/14062932/cc49440c-d053-4268-9d74-fe84848a82ab)

**Comment by simonsan** [comment]
This would be my proposal for now to be merged. If something better pops up, we can change easily.

![Logo_new_white](https://github.com/rustic-rs/rustic/assets/14062932/2f98faa0-0a71-4e2f-adab-e1bfadbe3790)
